tests/test_geometamaker.py

```diff
@@ -530,7 +540,7 @@ def test_describe_remote_datasource(self):
         """Test describe on a file at a public url."""
         import geometamaker
 
-        filepath = 'https://storage.googleapis.com/natcap-data-cache/global/aster-v3-1s/aster-v3-1s.tif'
+        filepath = 'https://storage.googleapis.com/releases.naturalcapitalproject.org/invest/3.14.2/data/CoastalBlueCarbon.zip'
```

Comment:
This is a smaller dataset; the test runs much faster. And maybe it's even more permanent too.